eras/conway/impl/test/data/conway-genesis.json

```diff
@@ -0,0 +1,52 @@
+{
+  "upgradeProtocolParams":{
+     "poolVotingThresholds":{
+        "pvtCommitteeNormal":0,
+        "pvtCommitteeNoConfidence":0,
+        "pvtHardForkInitiation":0,
+        "pvtMotionNoConfidence":0
+     },
+     "dRepVotingThresholds":{
+        "dvtMotionNoConfidence":0,
+        "dvtCommitteeNormal":0,
+        "dvtCommitteeNoConfidence":0,
+        "dvtUpdateToConstitution":0,
+        "dvtHardForkInitiation":0,
+        "dvtPPNetworkGroup":0,
+        "dvtPPEconomicGroup":0,
+        "dvtPPTechnicalGroup":0,
+        "dvtPPGovGroup":0,
+        "dvtTreasuryWithdrawal":0
+     },
+     "minCommitteeSize":0,
+     "committeeTermLimit":0,
+     "govActionExpiration":0,
+     "govActionDeposit":0,
+     "dRepDeposit":0,
+     "dRepActivity":0
+  },
+  "constitution": {
+    "anchor": {
+      "url": "",
+      "dataHash": "0000000000000000000000000000000000000000000000000000000000000000"
+    }
+  },
+  "committee" :{
+    "members" : [
```

Comment:
This is ugly - it's because of the default `ToJSONKey (Credential kr c)` implementation. 
Not sure how to change it, if we only use the hash as a key, we won't be able to tell the keyhash from the scripthash

Reply:
Looks like we need to solve this issue for Conway: https://github.com/input-output-hk/cardano-ledger/issues/3580

Reply:
I implemented something like this initially : 

```
instance Crypto c => ToJSONKey (Credential kr c) where
  toJSONKey = toJSONKeyText render
    where
      render = \case
        ScriptHashObj (ScriptHash hash) -> hashToTextAsHex hash
        KeyHashObj (KeyHash hash) -> hashToTextAsHex hash
``` and analogous FromJSONKey 

but then i realized we need to distinguish between ScriptHash and KeyHash representation no?

Reply:
If you want to tackle that ticket, I can tell you how to do it properly :wink:

Reply:
FYI, feel free to ignore this problem for this PR. It's technically not wrong what you have in this PR, it's just really ugly and we do want to fix it.